Sources/Card/Models/CardOrder.swift

```diff
@@ -0,0 +1,19 @@
+import Foundation
+
+public struct CardOrder {
+
+    /// The ID of the order to be approved
+    public let orderID: String
+
+    /// The card to be charged for this order
+    public let card: Card
+
+    /// Creates an instance of a card request
```

Comment:
```suggestion
    /// Creates an instance of a CardOrder
```

Should this still reference `card request`? Also, I'm curious why on Android we went with `CardRequest` and on iOS went with `CardOrder`?

Reply:
So on Android it's still `CardRequest`. I think the API for iOS will take the lead with naming in the long run, since they have really well thought out [design guidelines](https://www.swift.org/documentation/api-design-guidelines/) for Swift. We haven't explicitly said we're going to follow these for this SDK, but it would produce an API that feels most like existing Apple APIs.

Reply:
Also TLDR the plan was to finalize on iOS and circle back to Android.

Reply:
I went ahead and reverted. I think it'll help to re-visit once we have more merchant-facing features / sample integrations.